### DISCUSSION TITLE
How to convert ogg to mp3?

### DISCUSSION THREAD

**Discussion by kolisnichenko2828** [kokorin/Jaffree#387] [Q&A] [answered]
How to convert ogg to mp3?

**Comment by kokorin** [comment, marked as answer]
I would recommend you checking ffmpeg documentation first. The command is very simple `ffmpeg -i input.ogg output.mp3`.
See whole example (with progress tracking) [here](https://github.com/kokorin/Jaffree/blob/master/src/test/java/examples/ReEncodeExample.java).

```java
FFmpeg.atPath()
  .addInput(UrlInput.fromUrl(pathToOgg))
  .setOverwriteOutput(true)
  .addOutput(UrlOutput.fromUrl(pathToMp3))
  .execute();
```
